From 9e6b19a66d9b6b94395478fe79c5a3ccba181ad3 Mon Sep 17 00:00:00 2001
Message-Id: <9e6b19a66d9b6b94395478fe79c5a3ccba181ad3.1648731328.git.geliang.tang@suse.com>
From: Leon Huayra <hffilwlqm@gmail.com>
Date: Wed, 29 Dec 2021 22:44:22 +0800
Subject: [PATCH] bpf: Fix typo in a comment in bpf lpm_trie.
Git-commit: 9e6b19a66d9b6b94395478fe79c5a3ccba181ad3
Patch-mainline: v5.17-rc1
References: bsn#127
Modified-by-SEL: No


Fix typo in a comment in trie_update_elem().

Signed-off-by: Leon Huayra <hffilwlqm@gmail.com>
Signed-off-by: Alexei Starovoitov <ast@kernel.org>
Link: https://lore.kernel.org/bpf/20211229144422.70339-1-hffilwlqm@gmail.com
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 kernel/bpf/lpm_trie.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/bpf/lpm_trie.c b/kernel/bpf/lpm_trie.c
index 423549d2c52e..5763cc7ac4f1 100644
--- a/kernel/bpf/lpm_trie.c
+++ b/kernel/bpf/lpm_trie.c
@@ -412,7 +412,7 @@ static int trie_update_elem(struct bpf_map *map,
 		rcu_assign_pointer(im_node->child[1], node);
 	}
 
-	/* Finally, assign the intermediate node to the determined spot */
+	/* Finally, assign the intermediate node to the determined slot */
 	rcu_assign_pointer(*slot, im_node);
 
 out:
-- 
2.34.1

